src/libraries/System.IO.FileSystem/src/System/IO/EnumerationOptions.cs

```diff
@@ -27,13 +29,16 @@ public class EnumerationOptions
         /// </summary>
         internal static EnumerationOptions Default { get; } = new EnumerationOptions();
 
+        internal const int DefaultMaxRecursionDepth = int.MaxValue;
```

Comment:
Can you please move this constant to the beginning of the class (line 15)?

Reply:
Done.